day02/pam-main.sh

```
@@ -0,0 +1,195 @@
+# Author: Bhikesh Khute
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+YELLOW='\033[1;33m'
+WHITE='\033[1;37m'
+PURPLE='\033[0;35m' 
+CYAN='\033[0;36m'
+NC='\033[0m'
+echo "------------------------------------------------------"
+echo -e "=======\033[45m Welcome to Login Management System \033[0m======="
+echo "------------------------------------------------------"
+while true; do
```

Shellcheck (0.10.0)

[error] Couldn't parse this while loop. Fix to allow more checks.

(SC1073)